scripts/update-versions.mjs

```
@@ -0,0 +1,66 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// IMPORTANT: this script is intended to run as part of a GitHub workflow which is not installing the dependencies of the project.
+// So please, do not import code that is not provided by the node runtime. Otherwise, update the GitHub workflow definition
+import { readFileSync, writeFileSync } from 'node:fs';
+
+// run from the root of the repository: node scripts/update-versions.mjs 0.12.0.alpha-1
+const newVersion = process.argv[2];
```

ESLint

[error] 'process' is not defined.

(no-undef)